docs/getting-started.md

```diff
@@ -341,10 +340,10 @@ Now run `npm start` again and open http://localhost:8080. You should see a “he
 
 ### Step 4: Include JavaScript for a component
 
-Now that you have webpack configured to compile ES2015 into JavaScript, let's include the ES2015 files from the Material Design ripple. First install the Node dependency:
+Now that you have webpack configured to compile ES2015 into JavaScript, let's include the ES2015 files from the Material Design ripple. First, install the Node dependency:
 
 ```
-npm install --save-dev @material/ripple
```

Comment:
This change should be reverted. --save-dev saves @material/ripple as a dev dependency, which is need to compile the SCSS and JS.